rfcs/react-components/convergence/custom-styling.md

```diff
@@ -0,0 +1,214 @@
+# RFC: Custom Styling
+
+Geoff Cox
+Ben Howell
+Mason Tejera
+Brandon Thomas
+
+## Summary
+
+Fluent UI React v9 is built today to enforce adopting the Fluent 2 design styles. Partners creating component
+libraries don't know how provide their customers the ability to deviate from Fluent 2 design like they could
+with v0, v7, or v8.
+
+We need to define a custom styling mechanism or pattern for these partners.
```

Comment:
From what i've seen, it's not just our partner's customers...it's our own customers. Product X has different brand colors, so they tweak the theme. But those theme tweaks don't give them the colors they want/need for the Checkbox, so they need to manually adjust the Checkbox for their app.